app/(dashboard)/mailboxes/[mailbox_slug]/[category]/list/conversationFilters.tsx

```diff
@@ -78,17 +81,40 @@ export const useConversationFilters = () => {
     debouncedSetFilters(updates);
   };
 
+  const clearFilters = () => {
+    const clearedFilters = {
+      assignee: null,
+      createdAfter: null,
+      createdBefore: null,
+      repliedBy: null,
+      customer: null,
+      isVip: null,
+      isPrompt: null,
+      reactionType: null,
+      events: null,
+    };
+    setSearchParams((prev) => ({ ...prev, ...clearedFilters }));
```

Comment:
I think we should actually remove these params, if they still exist but are set to null it's more likely to create weird edge cases. So something like

```suggestion
    setSearchParams(({
      assignee,
      createdAfter,
      createdBefore,
      repliedBy,
      customer,
      isVip,
      isPrompt,
      reactionType,
      events,
      ...rest
    }) => rest);
```

Reply:
Oh I understand now, makes sense!